CONTRIBUTING.md

```diff
@@ -1,82 +1,79 @@
 
-First and foremost, Welcome to the **Lawnchair Launcher** Contributing Guidelines! This document will contains everything you'll need to contribute to the Lawnchair Launcher project.
+First and foremost, welcome to the **Lawnchair Launcher** Contributing Guidelines! This document will contain everything you'll need to contribute to the `Lawnchair Launcher` project.
 
 ## Before you start!
 
-We would like for you to be familiar with the tool we are using, the tool we are using will depend on which types of contribution you are going to make.
+We would like you to be familiar with the tools we are using. The tool(s) you will be using depend on the type of contribution you are going to make.
 
 ### Translation
 
-For translation, you would have to know only one tool: [Crowdin](https://lawnchair.crowdin.com) which allows us to collaborate with multiple translators & proofreaders.
+For translation, you only need to know one tool, [Crowdin](https://lawnchair.crowdin.com), which allows us to collaborate with multiple translators & proofreaders.
 
-Here are few tips for starter:
+Here are few tips for starters:
 
-- When using quotation marks, insert the symbols specific to the target language, as listed in this [this summary table](https://en.wikipedia.org/wiki/Quotation_mark#Summary_table);
-  
-- Lawnchair uses title case for some English UI text, Title case isn't used in other languages; opt for sentence case instead;
-  
-- Some English terminology may have no commonly used equivalents in other languages. In such cases, use short descriptive phrases—for example, the equivalent of *bottom row* for *dock*;
-  
-- Some languages *(e.g. French)* might have gender-specific word for masculine and feminine; recommend to opt for neutral-specific instead.
-  
+-   When using quotation marks, insert the symbols specific to the target language, as listed in this [this summary table](https://en.wikipedia.org/wiki/Quotation_mark#Summary_table);
+
+-   Lawnchair uses title case for some English UI text while other languages do not use title case; opt for sentence case instead;
+
+-   Some English terminology may have not have commonly-used equivalents in other languages. In such cases, use short descriptive phrases—for example, the equivalent of `bottom row` for `dock`;
+
+-   Some languages (e.g. `French`) has variations of words depending if it's masculine or feminine (gender-specific); we recommend to opt for gender-neutral words instead.
 
 ### Code
 
-For code, It's highly recommend that you use an Integrated Development Enviroment (IDE), know Java or preferably Kotlin and Git, which is a powerful Source Management Control for collaborating with multiple programmers.
+For code, it's highly recommended that you use an `Integrated Development Environment` (`IDE`), know `Java` (or preferably `Kotlin`), and `Git` which is a powerful source management control tool for collaborating with multiple programmers.
```

Comment:
Java, Kotlin and Git refers to Programming Languages and Source Management Control tool, should not be marked as codeblock.

External link to more information regarding these would be great

```suggestion
For code, it's highly recommended that you use an IDE like Android Studio, know Java (or preferably Kotlin), and Git which is a powerful source management control tool for collaborating with multiple programmers.
```